docker-compose.yml

```diff
@@ -1,32 +1,26 @@
 name: movies
 
-endpoints:
-  - service: frontend
-    path: /
-    port: 80
-  - service: api
-    path: /api
-    port: 8080
-
 services:
 
   frontend:
+    scale: 8
     build: frontend
     ports:
-      - 80
+      - 80:80
```

Comment:
this should be `80`. that way, it's not exposed publicly. the only public endpoint should be for a new `nginx` service in the compose file